### DISCUSSION TITLE
Secure Field Styling

### DISCUSSION THREAD

**Discussion by epcgrs** [mercadopago/sdk-react#87] [Q&A] [answered]
Is there any way to style the secure fields inputs?
like CardNumber?
I tried to add a className property but the component won't let me.

**Comment by AdaltonLeite** [comment, marked as answer]
Hi @epcgrs , 

o make a visual customization in Secure fields you must use the `style` property and not `className`. You can see how to use it in this [example](https://github.com/mercadopago/sdk-react/blob/main/examples/secureFields/CardNumber/0-Simple.tsx#L12). 

Below is a list of properties that can be customized:

|                    Property                    |
| :--------------------------------------------: |
|                    `color`                     |
|       `"font-family"` \| \| `fontFamily`       |
|         `"font-size"` \| \| `fontSize`         |
|        `"font-style"` \| \| `fontStyle`        |
|      `"font-variant"` \| \| `fontVariant`      |
|       `"font-weight"` \| \| `fontWeight`       |
|                    `height`                    |
|                    `margin`                    |
|     `"margin-bottom"` \| \| `marginBottom`     |
|       `"margin-left"` \| \| `marginLeft`       |
|      `"margin-right"` \| \| `marginRight`      |
|        `"margin-top"` \| \| `marginTop`        |
|                   `padding`                    |
|    `"padding-bottom"` \| \| `paddingBottom`    |
|      `"padding-left"` \| \| `paddingLeft`      |
|     `"padding-right"` \| \| `paddingRight`     |
|       `"padding-top"` \| \| `paddingTop`       |
| `"placeholder-color"` \| \| `placeholderColor` |
|        `"text-align"` \| \| `textAlign`        |
|                    `width`                     |

**Comment by epcgrs** [comment]
@AdaltonLeite thaks
